src/chttpd/src/chttpd_misc.erl

```diff
@@ -294,11 +296,25 @@ dbs_info_callback({error, Reason}, #vacc{resp = Resp0} = Acc) ->
 
 handle_task_status_req(#httpd{method='GET'}=Req) ->
     ok = chttpd:verify_is_server_admin(Req),
-    {Replies, _BadNodes} = gen_server:multi_call(couch_task_status, all),
-    Response = lists:flatmap(fun({Node, Tasks}) ->
-        [{[{node,Node} | Task]} || Task <- Tasks]
-    end, Replies),
-    send_json(Req, lists:sort(Response));
+    Jobs = lists:foldl(fun(Type, JobsAcc) ->
```

Comment:
It would be better to move this into `fabric2_fdb`. `fabric2_fdb:get_active_tasks` or something like that